### PR TITLE
Mccalluc/svg preview

### PR DESCRIPTION
**Pull request by mccalluc**
## Description

What was changed in this pull request? 
- index.html has been split vertically: The top half is still interactive, but the bottom half is the SVG export.
- The viewconf has been pulled into a standalone file for readability.

Why is it necessary?
- This will make SVG development easier by saving several clicks.
- This will make SVG development a higher priority, by making the differences more obvious.
- Trying out a different viewconf is easier: Just replace the file, instead of having to paste into the javascript.

(If you have thoughts on the TODOs, that is certainly welcome.)

## Checklist

- [ ] Unit tests added or updated
- [ ] Documentation added or updated
- [ ] Example added or updated
- [ ] Screenshot for visual changes (e.g. new tracks)

### CODE REVIEW

**Comment by mccalluc** [comment]
(Noticed #386, so I guess there is nothing like an `onLoad` right now. Good to know.)

**Comment by mccalluc** [comment]
Note how you can copy and paste view conf URLs after the `?`: Yes, you could edit the html file itself, but this is easier. You could have multiple viewconfs in different browser windows, without having to tweak multiple html files.

screen shot 2018-11-15 at 9 34 43 pm

Since HiGlass itself tends to be horizontal, I thought it would be more clear to stack these vertically. I think I was also thinking of it more as a side bar ... but the orientation isn't critical to me.

**Comment by pkerpedjiev** [comment]
Vertical stacking makes a lot more sense after looking at the screenshot. Carry on.

**Comment by mccalluc** [comment]
"carry on" = "✔️"?

**Comment by pkerpedjiev** [comment]
I approved the changes but there's still the outstanding comment about using the `on('location')` api call. That might get the SVG to update as the user pans around.

File: app/index.html [hunk 1]
```diff
@@ -9,415 +9,51 @@
...
+      document.getElementById('preview').innerHTML = hgApi.exportAsSvg();
+    }, 1000);
+    // Issue #386 is for adding an "onLoad" event, which would be better than this.
+  }
+  renderPreview();
 
-hgApi.on('view', () => {
-  console.log('View Config changed');
-});
+  // TODO: This does not seem to work: Is there a different event we should register?
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply app/index.html]: Neither callback works for me. I've reverted to develop, and don't see anything there, either. I've just filed #411... but I may just misunderstand the desired functionality.

**Pull request merged**